src/java/org/apache/cassandra/service/accord/AccordFastPath.java

```diff
@@ -194,7 +203,7 @@ public AccordFastPath withNodeStatusSince(Node.Id node, Status status, long upda
         }
 
         if (!canUpdateNodeTo(current, status, updateTimeMillis, updateDelayMillis))
-            throw new InvalidRequestException(String.format("cannot transition %s to %s at %s", node, status, updateTimeMillis));
+            throw new InvalidRequestException(String.format("cannot transition %s to %s at %s; current %s", node, status, updateTimeMillis, current));
```

Comment:
kept seeing this but missing the "why" was missing.  This didn't cause test failures, but when monitoring epochs I would see failed transformations leading me to debug them... just showing that we are trying to transition from status `A` to `A` isn't allowed is enough to know "I can ignore this"